virttest/utils_disk.py

```diff
@@ -18,6 +18,7 @@
     import ConfigParser
 from functools import cmp_to_key
 
+from aexpect import ShellCmdError, ShellStatusError
```

Comment:
Unused ShellCmdError and ShellStatusError imported from aexpect, that's should be a reason for ci check failure.

Reply:
thanks, have updated it.